UserAdminBundle/Controller/Admin/UserController.php

```diff
@@ -71,4 +72,37 @@ public function formAction(Request $request, $userId)
 
         return $this->renderAdminForm($form);
     }
+
+    /**
+     * @param Request $request
+     * @param string $userId
+     *
+     * @Config\Route("/password/change/{userId}", name="open_orchestra_user_admin_user_change_password")
+     * @Config\Method({"GET", "POST"})
+     *
+     * @Config\Security("has_role('ROLE_ACCESS_USER')")
+     *
+     * @return Response
+     */
+
+    public function changePasswordAction(Request $request, $userId)
+    {
+        /* @var User $user */
+        $user = $this->get('open_orchestra_user.repository.user')->find($userId);
+        $form = $this->createForm('user_change_password', $user, array(
+            'action' => $this->generateUrl('open_orchestra_user_admin_user_change_password', array('userId' => $userId))
+        ));
+        $form->handleRequest($request);
+
+        if ($form->isValid()) {
+            $userManager = $this->get('fos_user.user_manager');
+            $userManager->updateUser($user, true);
```

Comment:
you should not let the manager du the flush, it should be done once it for all at the end of the process